pages/browse/index.vue

```diff
@@ -2,12 +2,53 @@
 const { t } = useI18n();
 toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.browse"));
 
-const { data: playlists } = useCuratedPlaylists();
+const { data: browseSections, pending } = await useBrowse();
 </script>
 
 <template>
   <div>
     <PageHeading class="mb-6">{{ $t("nav.browse") }}</PageHeading>
-    <PlaylistOverview v-if="playlists" :playlists="playlists" />
+    <template v-if="pending">
+      <ul>
+        <li
+          v-for="index in 5"
+          :key="index"
+          class="my-6 h-11 w-full animate-pulse rounded-lg bg-background-2"
+        ></li>
+      </ul>
+    </template>
+    <ul v-else>
+      <template v-for="section in browseSections" :key="section.id || 0">
+        <PageHeading v-if="section.type === 'section_header'" :level="3">
+          <div class="flex items-center justify-between">
+            <div>
+              <NuxtLink
+                v-if="typeof section.link === 'string'"
+                :to="section.link as any"
+              >
+                {{ section.title }}
+              </NuxtLink>
+              <span v-else>{{ section.title }}</span>
+            </div>
+            <a v-if="section.link" :href="section.link">
+              <ButtonStyled intent="secondary" size="small">
+                <span class="whitespace-nowrap">
+                  {{ t("home.list.see-all") }}
+                </span>
+              </ButtonStyled>
+            </a>
+          </div>
+        </PageHeading>
+        <AlbumItem v-else-if="section.type === 'album'" :album="section" />
+        <PlaylistItem
+          v-else-if="section.type === 'playlist'"
+          :playlist="section"
+        />
+        <PodcastItem
+          v-else-if="section.type === 'podcast'"
+          :podcast="section"
+        />
+      </template>
```

Comment:
Please add a v-else which marks the other possible cases as red (not implemented) or something, or make somehow sure that we'll be informed by type hinting or similar that there is no other possible case. If you want to leave this up to me, please add a `Todo` in the code.